src/renderer/components/+namespaces/namespace-select.scss

```diff
@@ -4,19 +4,43 @@
   }
 }
 
+.GradientControl {
```

Comment:
Isn't it should be `.GradientValueContainer`? I can't find places of usages this classname in PR.
Should be a top-level classname for our component that overrides `react-select/components`? Otherwise I would put inside `.NamespaceSelect {}`

Reply:
Yes it should be and the use is here:

https://github.com/lensapp/lens/pull/2276/files#diff-3aed90c7f14be825f2f4c2eb5fb9031134438a51e096f40625c7d8b5f2a5d66cR80